cmd/edit/cluster/cmd.go

```diff
@@ -34,12 +34,17 @@ import (
 	"github.com/openshift/rosa/pkg/rosa"
 )
 
-const doubleQuotesToRemove = "\"\""
```

Comment:
@ciaranRoche I see we have quite a lot of this being used throughout the CLI. Wondering if we define a `pkg/inputs/consts.go` with this in it and then make everywhere us that?